pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -798,7 +803,8 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
     let guestAgentConfig: Record<string, any>;
     const enableKubernetes = K3sHelper.requiresPortForwardingFix(kubeVersion);
 
-    const privilegedServiceEnabled = await this.invokePrivilegedService('start');
+    const rdNetworking = !!cfg?.experimental.virtualMachine.networkingTunnel;
+    const privilegedServiceEnabled = rdNetworking && await this.invokePrivilegedService('start');
```

Comment:
This (as you noted) is backwards — if `rdNetworking` is enabled, then we do _not_ want to set `privilegedServiceEnabled`.

Reply:
Yes, that got in there by mistake :|